wordpress-mobile/WordPress-Android
# Recommended blogs card item in Discovery tab

Partially fixes: #12834

This PR adds a new item to the Discovery tab - "Recommended blogs". 
Following the blog will be done in a separate PR. 

Screenshots

Portrait
|  Light | Dark  |
|---|---|
| ![Screenshot_20200910-212344](https://user-images.githubusercontent.com/12660801/92791245-d0217700-f3ac-11ea-9c3c-8f2601f2aee5.jpg) | ![Screenshot_20200910-212435](https://user-images.githubusercontent.com/12660801/92791305-da437580-f3ac-11ea-9895-80f0cf812d01.jpg) |

Landscape
|  Light | Dark  |
|---|---|
| ![Screenshot_20200910-212357](https://user-images.githubusercontent.com/12660801/92791360-e3ccdd80-f3ac-11ea-962e-05d04c83a706.jpg) | ![Screenshot_20200910-212445](https://user-images.githubusercontent.com/12660801/92791397-eaf3eb80-f3ac-11ea-9022-8beed5bb7a7e.jpg) |

GIF
![recommended_blogs](https://user-images.githubusercontent.com/12660801/92791702-2d1d2d00-f3ad-11ea-8023-a2537119364c.gif)


To test:
- Go to the Discovery tab
- Scroll down to see "Sites to follow"

PR submission checklist:
